docs/rules/camelcase.md

```diff
@@ -70,6 +73,16 @@ var obj = {
 };
 ```
 
+### ignorePattern
+
+Examples of correct code for this rule with the `{ "ignorePattern": true }` option:
+
+```js
+/*eslint camelcase: ["error", {"ignoreStrings": ["ignored_.*"]}]*/
```

Comment:
Hi @not-an-aardvark , thanks for your comment.
I'm sorry for the confusion, It was a mistake. I fixed docs and add a commit.
https://github.com/fnwiya/eslint/commit/e1cae2adfe66398e28493c78d7e6e5955538a9ce